xpk.py

```diff
@@ -2764,16 +2755,17 @@ def delete_cluster_subnets(args) -> int:
   if return_code > 0:
     xpk_print('Listing all subnets failed!')
     return return_code
-  for index in range(1, 5):
+
+  for index in range(1, 9):
```

Comment:
Does this need to be 1,5 for a3 scenario and 1,9 for a3+?

Reply:
Nice catch! Based on offline discussion, I updated get_all_subnets_programmatic() to get more accurate results for subnets, so that we don't need to hardcode the number of subnets here. I also cleaned up the code based on pylint checks. PTAL.